bundles/org.openhab.ui/web/src/components/config/controls/item-picker.vue

```diff
@@ -10,7 +10,8 @@
       <f7-button slot="media" icon-f7="list_bullet_indent" @click.native="pickFromModel" />
     </f7-list-item>
     <!-- for placeholder purposes before items are loaded -->
-    <f7-list-item link v-show="!ready" :title="title">
+    <f7-list-item link v-else title="Loading...">
+      <f7-preloader />
```

Comment:
Please remove this preloader. It changes the list item's height in some cases and really doesn't look good - it's too much "in your face" esp. when you have several of them in a config sheet. Applying the disabled style and removing the chevron conveys properly, and in a subtle way, that the control is not ready yet.

Reply:
I assumed you wouldn't like the preloader too much 😄.
Though, I think your suggestion is a little too "subtle" - it doesn't indicate to the user that something's going to change, does it? You really don't even want to change the title to "Loading..."? IMHO there should be at least a tiny something telling the user that the page is not ready yet.

Reply:
Here's my take on it: the item picker is usually ready very quickly, maybe tenths of a second, and in many cases _before_ the user even has a chance to click on it, so they're not even aware there was something loading in the background in the first place. I never really had a case of an item picker being opened so quickly after being shown that it wasn't ready.

On the other hand, having something flickering before your eyes is almost always a bad look (unless it's an important part of the page), and here we'd be having every item picker briefly and conspicuously showing its "loading" state when shown, unnecessarily drawing the user's attention. I'd rather have the occasional user click twice because it was not ready the first time than the flickering.

So I believe in this case the subtler, the better. And of course it will be a moot point when we have a Vuex cache of items with a reasonable strategy to keep it up-to-date behind the scenes, after the cache is populated every item picker will be ready instantaneously.